frontend/src/utils/checkValidCodes.ts

```diff
@@ -0,0 +1,21 @@
+const socPrefixes = ['00', '11', '13', '15', '17', '19', '21', '23', '25', '27', '29', '31', '33', '35', '37', '39', '41', '43', '45', '47', '49', '51', '53']
+
+// const cipPrefixes = ['01', '03', '04', '05', '09', '10', '11', '12', '13', '14', '15', '16', '19', '21', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47', '48', '49', '50', '51', '52', '53', '54', '55', '60', '61']
```

Comment:
Do we need this commented out code?

Reply:
I kept it in case we wanted a CIP validator and it was a lot of numbers to look up.